spec/models/reservation_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Reservation, type: :model do
+  before do
+    @user = User.create(email: 'a@b.c', username: 'user1', password: '123456', password_confirmation: '123456')
+    @destination = Destination.create(name: 'London', location: 'London', price_per_day: 25,
+                                      image_url: 'https://via.placeholder.com/150')
+  end
+
+  subject do
+    Reservation.create(user_id: @user.id, destination_id: @destination.id, startingDay: '01/01/2015',
+                       endingDay: '01/01/2015')
+  end
+
+  it 'create a valid destination object' do
+    expect(subject).to be_valid
+  end
+
```

Comment:
You don't need to make this mocking variables when you using shoulda gem. So you can remove this piece of code. It's unnecessary

Reply:
thanks @ShinobiWarior for your suggestion. I agree and have fixed it.